src/content/en/2021/accessibility.md

```diff
@@ -354,7 +354,7 @@ Tables should have a well-formatted structure with the appropriate elements and
 
 ### Table captions
 
-Table captions act as a heading for the full table to provide a summary of its information. When labelling a table, the `<caption>` element is the correct semantic choice to provide the most context to a screen reader user.
+Table captions act as a heading for the full table to provide a summary of its information. When labelling a table, the `<caption>` element is the correct semantic choice to provide the most context to a screen reader user, though it should be noted that there are also [other alternative captioning techniques for tables](https://www.w3.org/WAI/tutorials/tables/caption-summary/).
```

Comment:
We use a figcaption! So felt weird saying this just before a table without at least adding a comment to this effect.

Added an issue so we can test this method as well next year: https://github.com/HTTPArchive/legacy.httparchive.org/issues/248